iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -88,6 +88,11 @@ internal sealed class MqttTransportHandler : TransportHandler, IMqttIotHubEventH
         private const string ReceiveEventMessagePatternFilter = "devices/{0}/modules/{1}/#";
         private const string ReceiveEventMessagePrefixPattern = "devices/{0}/modules/{1}/";
 
+        // Identifiers for property update operations.
+        public const string VersionKey = "$version";
+
+        public const string RequestIdKey = "$rid";
```

Comment:
I'm surprised we didn't have this just for regular twin/method operations already. Were we never sending a requestId for those?

Reply:
Ah, nevermind. I see below that we just had it locally defined instead